lib/hanami/utils/string.rb

```diff
@@ -151,6 +158,24 @@ def classify
         self.class.new words.zip(delimiters).join
       end
 
+      # Return true if the string is a valid class name
+      #
+      # @return [Boolean]
+      #
+      # @since 1.0.x
+      #
+      # @example
+      #   require 'hanami/utils/string'
+      #
+      #   string = Hanami::Utils::String.new 'hanami_utils'
+      #   string.classified? # => false
+      #
+      #   string = Hanami::Utils::String.new 'HanamiUtils'
+      #   string.classified? # => true
+      def classified?
```

Comment:
If we decide to go with `#classified?` this should be a private method and `@api private`.